packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -218,6 +219,45 @@ export const GCNodeType = {
  */
 export type GCNodeType = (typeof GCNodeType)[keyof typeof GCNodeType];
 
+/**
+ * The contents of a garbage collection sweep message.
+ * @internal
+ */
+export interface ISweepMessageContents {
+	// The ids of nodes that are deleted.
+	deletedNodeIds: string[];
+}
+
+/**
+ * The content type of a garbage collection message content.
+ * @internal
+ */
+export type GarbageCollectionMessageContents = ISweepMessageContents;
+
+/**
+ * The const types of a garbage collection message.
+ * @internal
+ */
+export const GarbageCollectionMessageType = {
+	// Message that is sent after sweep runs.
+	Sweep: "Sweep",
+};
+/**
```

Comment:
In line with other suggested type changes below:

```suggestion
export const GarbageCollectionMessageType = {
	// Message that is sent after sweep runs.
	Sweep: "Sweep" as const,
} satisfies Record<string, GarbageCollectionMessageType>;
```

Reply:
I changed it as per your suggestion. However, I wish that I could use `GarbageCollectionMessageType.Sweep` instead of `"Sweep"`. when processing messages.